Gemfile

```diff
@@ -7,21 +7,15 @@ source "https://rubygems.org"
 #
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
-# gem "jekyll", "~> 4.3.2" # To run locally, uncomment this line.
-# This is the default theme for new Jekyll sites. You may change this to anything you like.
-gem "minima", "~> 2.5"
-# If you want to use GitHub Pages, remove the "gem "jekyll"" above and
+gem "jekyll", "~> 4.3.2"
+# If you want to use legacy GitHub Pages, remove the "gem "jekyll"" above and
 # uncomment the line below. To upgrade, run `bundle update github-pages`.
-gem "github-pages", "~> 228", group: :jekyll_plugins # To run locally, comment out this line.
+# gem "github-pages", "~> 228", group: :jekyll_plugins
```

Comment:
presumably this should be restored before merge?

Is there a better way to handle this for local testing?

Reply:
> Swaps out github-pages gem for jekyll, as the former is meant for legacy GitHub Pages deployments (which we aren't using) while the latter is more up to date.

This comment ^^^ makes me suspect we should be removing the reference to the github-pages gem.

Reply:
Keith is correct, `github-pages` is supposed to be commented out as we actually don't need it if we deploy using GitHub actions. The `github-pages` gem is a fork of Jekyll 1.x, which is very old. We want to be using the much newer `jekyll` gem both locally and on the web.

I left it in just in case we ever did want to use the legacy option, but I don't think that's very likely. I'm happy to ~~kill it with fire~~ remove it altogether 😉